pages/osx/say.md

```diff
@@ -1,19 +1,23 @@
 # say
 
-> Uses text-to-speech to speak through the default sound device.
+> Converts text to speech.
 
 - Speak a phrase aloud:
 
-`say "I like to ride my bike."`
+`say {{"I like to ride my bike."}}`
 
 - Speak a file aloud:
 
-`say -f {{filename}}`
+`say -f {{filename.txt}}`
 
-- Create an AAC compressed audio file with the spoken text:
+- Speak a phrase with a custom voice and speech rate:
 
-`say -o {{filename.m4a}} "Everyone loves iTunes"`
+`say -v {{voice}} -r {{words_per_minute}} {{"I'm sorry Dave, I can't let you do that."}}`
```

Comment:
That's the default, but in some cases IIRC it's been decided that an actual example is clearer. It boils down to what's best for the user, rather than following a guideline strictly. In this case I sadly agree that {{message}} is the most appropriate choice.